src/lipalightninglib.udl

```diff
@@ -40,6 +40,11 @@ interface LightningNode {
     [Throws=LnError]
     LspFee query_lsp_fee();
 
+    // Limits in the amount sats a payment is allowed to transfer. Currently there are only limits for receiving payments.
+    // The limits (partly) depend on the channel situation of the node, so it should be called again every time the user is about to receive a payment.
+    // The limits stay the same regardless of what amount the user enters that he wants to receive; they solely depend on the channel situation.
```

Comment:
```suggestion
    // The limits stay the same regardless of what amount the user wants to receive; they solely depend on the channel situation.
```

Reply:
I agree my phrase reads badly. But I purposefully wanted to write down "entering amount" because it was a FAQ, whether the limits change while the user is typing an amount.

Reply:
To me, it remains understandable, but we can keep it if you disagree.

Reply:
How about https://github.com/getlipa/lipa-lightning-lib/compare/b6fdc36f2795b358153e9087bece611928a69935..8bb0a178bb02b98e1518fd0aa7382316a2e287ea ?